### ISSUE TITLE
0 Improvement after 8M env steps finetuning on Transport

### ISSUE THREAD

**Issue by wkguo** [irom-lab/dppo#56]
Hi, following the settings in the original YAML file. I finetuned the **ppo_diffusion_mlp_pixel** policy based on the Robomimic-Transport task, starting from _transport_pre_diffusion_mlp_img_ta8_td100/checkpoint/state_2000. During the fine-tuning process, I found that the loss (overall and value) converges really quickly; however, the success rate remains 0 all the time (8M steps in total). 
This is abnormal in comparison to both your paper result and reproduce result in **ppo_diffusion_mlp_state** (its success rate keeps growing), do I miss some configurations? Or should the pre-training be longer?
Thanks for your effort!

**Comment by Duckyee728** [comment]
i met the same issue, do you solve it?
